### PR TITLE
update fix error region id is null

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

File: plugin/Region/RegionController.php [hunk 1]
```diff
@@ -100,6 +100,9 @@ private function createRegionObjectsRegionOne(array $regionList): array
 	{
 		$regions = [];
 		foreach ($regionList as $regionProperties) {
+			if (empty($regionProperties['elements']['id'])) {
```
Comment (jayay) [inline plugin/Region/RegionController.php:103]: `!isset()` would be preferred over `empty()`. That's a minor thing, though.

**Review by jayay — COMMENTED** [review]
Can this change be tested? That'd be good

File: plugin/Region/RegionController.php (hunk 1 above)
Comment (hungnc89) [reply plugin/Region/RegionController.php:103]: `empty()` includes checking that the variable exists and is not null. If the api returns id is null there will still be an error

**Comment by hungnc89** [comment]
> Can this change be tested? That'd be good

**Comment by hungnc89** [comment]
Hi @jayay,
I added test for the task

**Comment by jayay** [comment]
Thank you!

**Pull request closed without merging**